### ISSUE TITLE
Cannot open dashboard (error after login in logs, but no message on login page)

### ISSUE THREAD

**Issue by pointout** [louislam/uptime-kuma#5097]
### 📑 I have found these related issues/pull requests

didn't found one

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

No login possible. When entering and submitting the username and password, the login page is shown. No error message is shown on login page, but found this in the logs:

```
Trace: SyntaxError: Unexpected end of JSON input
  at JSON.parse (<anonymous>)
  at Proxy.toJSON (/home/cleavr/xxxxx/releases/20240910080404949/server/model/monitor.js:167:30)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async UptimeKumaServer.getMonitorJSONList (/home/cleavr/xxxxx/releases/20240910080404949/server/uptime-kuma-server.js:223:34)
  at async UptimeKumaServer.sendMonitorList (/home/cleavr/xxxxx/releases/20240910080404949/server/uptime-kuma-server.js:203:20)
  at async afterLogin (/home/cleavr/xxxxx/releases/20240910080404949/server/server.js:1668:23)
  at async Socket.<anonymous> (/home/cleavr/xxxxx/releases/20240910080404949/server/server.js:416:21)
  at process.unexpectedErrorHandler (/home/cleavr/xxxxx/releases/20240910080404949/server/server.js:1859:13)
  at process.emit (node:events:526:35)
  at emit (node:internal/process/promises:149:20)
  at processPromiseRejections (node:internal/process/promises:283:27)
  at process.processTicksAndRejections (node:internal/process/task_queues:96:32) 
```

### 👟 Reproduction steps

try to login

(alternative data directory is set)

### 👀 Expected behavior

seeing the dashboard

### 😓 Actual Behavior

only login page is shown. Public status page is working.

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Ubuntu 22.04

### 🌐 Browser

Firefox 131.0b2, MS Edge  128.0.2739.63

### 🖥️ Deployment Environment

- Runtime: nodejs 18, nginx proxy, PM2
- Database: mariadb
- Filesystem used to store the database on: 
- number of monitors:

**Comment by CommanderStorm** [comment]
> alternative data directory is set

Could you please share your configuration?

**Comment by pointout** [comment]
I am using cleavr.io for deployment. This is my PM2 config there:
```
module.exports = {
  name: "up.xxxx.at",
  script: "server/server.js",
  args: "",
  log_type: "json",
  cwd: "/home/cleavr/up.xxxx.at/current",
  // Note: We detected that this web app is running on a server with multiple CPUs and hence we are
  // setting *instances* to "max" and *exec_mode* to "cluster_mode" for better performance.

  instances : "1", // change the value to "1" if your server has only 1 CPU
  // exec_mode : "cluster_mode", // remove this line if your server has only 1 CPU
  env: {
    "UPTIME_KUMA_PORT": 8334,
    "DATA_DIR": "/home/cleavr/up.xxxx.at/data",
    "CI": 1,
    "NUXT_TELEMETRY_DISABLED": 1
  }
}
```
In all releases this cleavr config is linked.
I rolled back for 2 deployments to the deployment from 25.8.2024. This versions works and I can log in.

The Server is a Arm64 vServer. Maybe something changed, that's not working on Arm64.

**Comment by CommanderStorm** [comment]
> I rolled back for 2 deployments to the deployment from 25.8.2024

What do you mean by this part?

**Comment by pointout** [comment]
I have a github action that syncs my fork every sunday at 1pm. Everytime there is a code push to master, cleavr.io starts a deployment. My last working deployment is from 25th Aug.
So any commit between 25th Aug and 1st Sept is causing the problem in logging in. 

I did setup a freh uptime kuma installation from the same repository on the same server using sqlite. I could set a user and log in. Then I moved the data directory one level up and changed the setting accordingly. It worked.
Then I changed the data directory to the one of my old installation with the mariadb and I got the same error. There is something with my DB.
